java/api/src/main/java/io/ray/api/PyActorCall.java

```diff
@@ -40,4 +40,10 @@ default <R> PyActorTaskCaller<R> task(PyActorMethod<R> pyActorMethod, Object obj
     return new PyActorTaskCaller<>((PyActorHandle)this, pyActorMethod, args);
   }
 
+  default <R> PyActorTaskCaller<R> flatten_task(PyActorMethod<R> pyActorMethod, Object obj0, Object[] obj1) {
```

Comment:
Seems a new API ".flatten_task" is added here. Can we avoid api changes and fix this issue by changing the logic of the existing ".task" API ?

Reply:
@SongGuyang I have removed the new function. Kindly share your thoughts and suggestions with me.

Reply:
@SongGuyang have passed all tests.
https://buildkite.com/ray-project/premerge/builds/26727